src/page-modules/departures/departures.module.css

```diff
@@ -71,14 +71,15 @@
   width: 100%;
   max-width: var(--maxPageWidth);
   margin: 0 auto;
-  padding: 0 var(--spacings-xLarge) var(--spacings-xLarge) var(--spacings-xLarge);
+  padding: 0 var(--spacings-xLarge) var(--spacings-xLarge)
+    var(--spacings-xLarge);
   z-index: 10;
   position: absolute;
   left: 0;
   right: 0;
 }
 
-@media (max-width: 650px) {
+@media (max-width: 600px) {
```

Comment:
Choose to change the transition in the placement of the components from 650px to 600px as the transition flows better in my opinion.